src/main/steps/applicant1/review-the-application/content.ts

```diff
@@ -164,13 +164,22 @@ export const form: FormContent = {
   },
 };
 
+const getApplicant1PartnerContent = (content: CommonContent): string => {
+  if (content.userCase?.sameSex !== Checkbox.Checked && content.partner !== content.civilPartner) {
+    return content.partner === content.husband ? content.wife : content.husband;
+  } else {
+    return content.partner;
+  }
+};
+
```

Comment:
since this is also used on the check-your-joint-application page it's probably worth moving the function into content.utils.ts so that we can reuse it

Reply:
Also updated check-your-joint-application page to include getApplicant1PartnerContent function from common content.utils.